azurerm/resource_arm_front_door.go

```diff
@@ -172,24 +172,25 @@ func resourceArmFrontDoor() *schema.Resource {
 										Required:     true,
 										ValidateFunc: afd.ValidateBackendPoolRoutingRuleName,
 									},
+									// Remove default value for #4461
 									"cache_use_dynamic_compression": {
 										Type:     schema.TypeBool,
 										Optional: true,
-										Default:  false,
```

Comment:
I think this will end up causing people a diff?

Reply:
I don't think this will be an issue since it's data type is bool, it is always `false` by default... WDYT?